### PR TITLE
clio-33169 expose total record count in th-table

### PR DESCRIPTION
**Pull request by PANCAKES**

### CODE REVIEW

**Comment by JeffMarvin** [comment]
Looks good to me. Sending back your way @PANCAKES

File: themis_components/thTable/readme.md [hunk 1]
```diff
@@ -217,6 +217,9 @@ that you can use:
   * Reference to the TableHeader objects array that you pass to the delegate
     on creation.
 
+* `totalRecords`
+  * Returns the total number of records in the complete data set for the table.
+
```
Comment (seanhealy) [inline themis_components/thTable/readme.md]: Not really needed but I added this to the Pagination Example to show off this new feature while testing.

```html
    <p>Total Records: {{ demo.tableDelegate.totalRecords() }}</p>
    <p>Page Record Count: {{ demo.tableDelegate.getData().length }}</p>
```

Comment (seanhealy) [inline themis_components/thTable/readme.md]:
```diff
diff --git a/themis_components/thTable/examples/2 - Pagination/html.html b/themis_components/thTable/examples/2 - Pagination/html.html
index d8cea50..41bcc7b 100644
--- a/themis_components/thTable/examples/2 - Pagination/html.html	
+++ b/themis_components/thTable/examples/2 - Pagination/html.html	
@@ -10,5 +10,8 @@
         </th-table-cell>
       </th-table-row>
     </th-table>
+
+    <p>Total Records: {{ demo.tableDelegate.totalRecords() }}</p>
+    <p>Page Record Count: {{ demo.tableDelegate.getData().length }}</p>
   </body>
 </html>
```

Comment (seanhealy) [inline themis_components/thTable/readme.md]: :white_check_mark: Resolved

**Pull request merged**